docs/changelog.md

```diff
@@ -1,5 +1,15 @@
 # NEURON 8.2
 
+## 8.2.7
+_Release Date_ : 25-05-2025
+
+This release allows use of Python 3.13.
+Pins numpy to >=2.
```

Comment:
Do we need to use numpy >=2 at runtime? As far as I understood reading numpy docs, we so need to compile NEURON with >=2, but since its backwards compatible, we can use 1.x at runtime.

Reply:
Apparently 1.x is ok after installation.  I was successful with
```
$ python -m venv env313
$ source env313/bin/activate
$ pip install 'numpy<2'
$ pip install neuron
>>> import numpy
>>> numpy.__version__
'1.26.4'
>>> import neuron
>>> neuron.test_rxd()
...
Ran 4 tests in 1.537s

OK
True
>>> 
```
But 'pins numpy' can be taken to refer to the build process and I'm too exhausted from the release workflow to want anything more to do with 8.2.7.  By the way, I think we can eliminate the CircleCI Pipeline. 8.2.8 will have to adopt the master approach more fully.

Reply:
> I'm too exhausted from the release workflow to want anything more to do with 8.2.7

Fair enough, I agree the release workflow should be refactored so it involves fewer steps (and I have some ideas on how to achieve this, at least for v9 and onwards).